tasks/install-worker.yml

```diff
@@ -27,11 +27,14 @@
 
 - name: create worker service | concourse
   template:
-    src: concourse-worker.service.j2
-    dest: /etc/systemd/system/concourse-worker.service
+    src: "{{ item['src'] }}"
+    dest: "{{ item['dest'] }}"
     owner: root
     force: yes
   become: yes
   become_user: root
+  with_items:
```

Comment:
with_items could be revert ?

Reply:
Same reason as https://github.com/cycloidio/ansible-concourse/pull/5#discussion_r460779381